### ISSUE TITLE
Horizon and Telescope

### ISSUE THREAD

**Issue by drewkennelly** [laravel/telescope#101]
Nice work on the app, it works really well so far.

One (small) issue I've come across is that the Requests section gets flooded with data from Horizon every second, so it makes it very difficult to view Request data from my app.

This only happens when you have Horizon open in the browser as it makes GET XHR requests to it's API - which in turn get logged in Telescope.

Is there a way to filter out certain requests - specifically from Horizon?

Screenshot below.

<img width="1171" alt="screenshot 2018-10-24 17 24 42" src="https://user-images.githubusercontent.com/279851/47410548-db1b9000-d7b1-11e8-8750-3119f06d24bc.png">

**Comment by ultimateBusiness** [comment]
@themsaid - it would be great if we can have an option via an array in the config to exclude path's (for example: "horizon/*") from the request-watcher.

With Debug Bar it's the same issue.

**Comment by okaufmann** [comment]
Till then you could do something like this in your TelescopeServiceProvider's register method:

```php
Telescope::filter(function (IncomingEntry $entry) {

            if ($entry->type === EntryType::REQUEST
                && isset($entry->content['uri'])
                && str_contains($entry->content['uri'], 'horizon')) {
                return false;
            }

            if ($this->app->environment() == 'local') {
                return true;
            }

            // .....
    }
```

**Comment by VladimirGr** [comment]
As I understand it, the telescope is processing queues. Ie, if you used the horizon for bursts, now you can remove it and work only with a telescope. Or am I wrong?

> The jobs tab lists out all of the jobs that have run or all running. It's similar to Horizon, but Horizon is Redis-only and isn't just a UI, and it also interacts with how your queue workers are running. Telescope, on the other hand, is just a UI, but it also works for all queue drivers.

**Comment by thibaultlavoisey** [comment]
@okaufmann answer's is a good one, but it may be useful to also filter events, queries and so on.

**Comment by browner12** [comment]
@VladimirGr Telescope does not process the queue, you'll still need to keep Horizon for that.

**Comment by sifex** [comment]
@drewkennelly Fixed with #200 , totally agree that horizon should be hidden by default,
